src/main/java/com/dev/cinema/model/dto/moviesession/MovieSessionRequestDto.java

```diff
@@ -1,8 +1,17 @@
 package com.dev.cinema.model.dto.moviesession;
 
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
 public class MovieSessionRequestDto {
+    @NotNull(message = "The movieId couldn't be null")
+    @Min(value = 1, message = "Length should be longer then 1")
```

Comment:
If you apply min for Long or Integer fields does it check the length of value or the value itself?

Reply:
I see the point, when we use @Min check just value and when we use @Size(min = 4) in this case we check length